### PR TITLE
Merge fix for bug 1663251 nto 5.7

### PR DESCRIPTION
**Pull request by gl-sergei**
http://jenkins.percona.com/view/5.7/job/mysql-5.7-param/685/

### CODE REVIEW

**Review by laurynas-biveinis — APPROVED** [review]

**Comment by laurynas-biveinis** [comment]
This needs a second review from either @robgolebiowski or @percona-ysorokin

**Review by robgolebiowski — APPROVED** [review]
LGTM

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**
